src/riak_ensemble_peer.erl

```diff
@@ -2003,6 +2019,13 @@ existing_leader(_Replies, Abandoned, #fact{epoch=Epoch, seq=Seq, leader=Leader})
             undefined
     end.
 
+notify_leader_status(PidList, StateName, State) when is_list(PidList) ->
+    [notify_leader_status(P, StateName, State) || P <- PidList];
+notify_leader_status(Pid, leading, #state{id = Id, ensemble = Ensemble}) ->
+    Pid ! {is_leading, self(), Id, Ensemble};
```

Comment:
Sure, that seems reasonable. I can't really imagine the sequence number being too useful but I can see the epoch being useful since you can use it to track how many different leaders the ensemble has been through so far and such. It's not useful for the use case I'm looking at, but still probably best to add it now so that we don't have to break the API in the future if we want to add it. I'll rebase and push in a minute once I've implemented and tested the change.